modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosController.scala

```diff
@@ -56,7 +56,8 @@ object GmosController {
     @SuppressWarnings(Array("org.wartremover.warts.LeakingSealed"))
     case class BuiltInFPU(fpu: T#FPU) extends GmosFPU
 
-    case class GmosDisperser(disperser: T#Disperser, order: Option[DisperserOrder], lambda: Option[Length])
+    case class GmosDisperser(disperser: T#Disperser, order: Option[DisperserOrder],
+      lambda: Option[Length])
 
     case class CCConfig(
```

Comment:
It doesn't look like we're extending any of these, so why not make them `final case class`?

Reply:
Making them `final` produces a compile error: "The outer reference in this type test cannot be checked at run time."